src/cleanvision/imagelab.py

```diff
@@ -301,17 +440,73 @@ def visualize(
         examples_per_issue: int = 4,
         cell_size: Tuple[int, int] = (2, 2),
     ) -> None:
+        """Visualization helper for images.
+
+        This method is multipurpose and can be used for visualizaing:
+        1. Random images from the dataset
+        2. Particular images with paths given in image_files
+        3. Top examples of given issue_types found in the dataset
+
+        If no image_files or issue_types are given, random images will be shown from the dataset.
+        If specific image_files are given, it will override the argument issue_types and will show given image_files
+        If image_files are not given and issue_types are given, top examples of given image_types will be shown
+
+        Parameters
+        ----------
+
+        image_files : List[str], optional
+            List of image filepaths to visualize
+
+        issue_types: List[str], optional
+            List of issue types to visualize
+
+        num_images : int, optional
+            Number of images to randomly visualize from the dataset
+            Used only when image_files and issue_types are empty, otherwise this argument will be ignored
+
+        examples_per_issue : int, optional
+            Number of top examples per issue type to visualize
+            Used only when issue_types is given and image_files is empty
```

Comment:
what do you think about merging these two arguments to just be one argument: `num_images` ?

To me they have roughly the same effect (control how many images are plotted). In random case, it is the total number of images, otherwise it is the number of images per issue type.

Reply:
Yeah, I can merge them.